src/spec/tsconfig.json

```diff
@@ -0,0 +1,8 @@
+{
+    "compilerOptions": {
+        "out": "../dist/Excalibur.js", 
```

Comment:
It doesn't look like it does, I think it is because we have a custom grunt task. If you ran `tsc` in the directory it would. I really only added this tsconfig so that vscode wouldn't complain about the decorators.